### PR TITLE
Benji/ios 564 add typing indicator

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: Shared/UI/Views/MessageContentView/MessageContentView.swift [hunk 1]
```diff
@@ -251,7 +246,13 @@ extension MessageTextView {
     }
 
     /// Updates the font size to be appropriate for the amount of text displayed.
-    fileprivate func updateFontSize() {
+    fileprivate func updateFontSize(state: MessageContentView.Layout) {
+        
+        if state == .collapsed {
+            self.font = FontType.regular.font
```
Comment (Benjamindodgson) [inline Shared/UI/Views/MessageContentView/MessageContentView.swift:252]: Fixes issue with large font in the collapsed state.

File: iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift [hunk 2]
```diff
@@ -156,9 +156,7 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
     }
 
     private func configureCollectionLayout(for state: ConversationUIState) {
-        self.collectionLayout.itemHeight
-        = MessageContentView.bubbleHeight + old_MessageDetailView.height + Theme.ContentOffset.short.value
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift:160]: Moved to the layout

File: iOS/Flows/Conversation/Input/TypingIndicatorView.swift [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+import Foundation
+import Combine
+import StreamChat
+
+class TypingIndicatorView: BaseView {
+    
+    private let label = ThemeLabel(font: .small)
+    
+    var cancellables = Set<AnyCancellable>()
+    var subscriptions = Set<AnyCancellable>()
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Input/TypingIndicatorView.swift:18]: Here's the typing logic.

File: iOS/UI/Shared/StackedPersonView.swift [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    private var peopleIds: [String] = []
+    
+    override func initializeSubviews() {
+        super.initializeSubviews()
+        
+        self.clipsToBounds = false
+    }
+    
+    func configure(with people: [PersonType]) {
+        let ids = people.compactMap { person in
```
Comment (Benjamindodgson) [inline iOS/UI/Shared/StackedPersonView.swift:23]: Little optimization so we don't constantly layout for the same people.

**Review by MartinJibber — APPROVED** [review]

**Pull request merged**